app/lib/webpack/loader.auto-import.js

```diff
@@ -47,7 +47,12 @@ function extract (content, form) {
 }
 
 module.exports = function (content) {
-  if (!this.resourceQuery) {
+  // regex to match functional components
+  let funcCompRegex = new RegExp(
```

Comment:
Please change this to:

```javascript
const funcCompRegex = /var\s+component\s*=\s*normalizer\((?:[^,]+,){3}\s*true,/
```

and move it on to, outside the function (after const dirRegex)